homework13/hw13.py

```diff
@@ -0,0 +1,88 @@
+
+#2
+def new_format(string):
+    format_string = ("{:,}".format(int(string)).replace(',', '.'))
+    return format_string
+
+
+assert (new_format("1000000") == "1.000.000")
+assert (new_format("100") == "100")
+assert (new_format("1000") == "1.000")
+assert (new_format("100000") == "100.000")
+assert (new_format("10000") == "10.000")
+assert (new_format("0") == "0")
+print('Tests Done')
+
+#1 If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of
+# these multiples is 23.
+#
+# Finish the solution so that it returns the sum of all the multiples of 3 or 5 below the number passed in.
+#
+# Additionally, if the number is negative, return 0.
+#
+# Note: If the number is a multiple of both 3 and 5, only count it once.
+
+
+def solution(number):
+    sum = 0
+    for i in range(2, number):
+        if ((i % 3 == 0) or (i % 5 == 0)):
+            sum += i
+    return sum
+
+#2 In this kata you will create a function that takes a list of non-negative integers and strings and returns a new list
+# with the strings filtered out.
+# filter_list([1,2,'a','b']) == [1,2]
+# filter_list([1,'a','b',0,15]) == [1,0,15]
+# filter_list([1,2,'aasf','1','123',123]) == [1,2,123]
+
+
+def filter_list(l):
+    return [x for x in l if isinstance(x, int)]
+
+#3
+# Your task is to make a function that can take any non-negative integer as an argument and return it with its digits in
+# descending order. Essentially, rearrange the digits to create the highest possible number.
+
+
+def descending_order(num):
+    num = str(num)
+    desc_ord = ''.join(sorted(num, reverse=True))

```

Comment:
bad naming
desc_ord -> desc_order
but here you can make it in one line. Remove useless variables